change-notes/1.20/analysis-cpp.md

```diff
@@ -19,6 +19,9 @@
 | Lossy function result cast (`cpp/lossy-function-result-cast`) | Fewer false positive results | The whitelist of rounding functions built into this query has been expanded. |
 | Unused static variable (`cpp/unused-static-variable`) | Fewer false positive results | Variables with the attribute `unused` are now excluded from the query. |
 | Resource not released in destructor (`cpp/resource-not-released-in-destructor`) | Fewer false positive results | Fix false positives where a resource is released via a virtual method call, function pointer, or lambda. |
+| 'new[]' array freed with 'delete' | More correct results | Data flow through global variables for this query has been improved. |
+| 'new' object freed with 'delete[]' | More correct results | Data flow through global variables for this query has been improved. |
+| Mismatching new/free or malloc/delete | More correct results | Data flow through global variables for this query has been improved. |
```

Comment:
Please add query ids to these entries so Felicity won't have to reconstruct them right before the release.

Reply:
Done.